src/create-render-context.js

```diff
@@ -208,6 +177,16 @@ const patchTimers = (): void => {
     patchCallbackFnWithGate(window, "requestAnimationFrame", "__SSR_ACTIVE__");
 };
 
+const createVirtualConsole = () => {
+    const virtualConsole = new VirtualConsole();
+    virtualConsole.on("jsdomError", (e: Error) => {
```

Comment:
Ah! So errors will get bubbled up but we'll ignore JSDOM-specific errors, this is really elegant, thank you!

Reply:
So, we're not ignoring them yet...but we can now ignore the image ones. (see my latest push)